modules/api/src/main/java/io/fluo/api/config/FluoConfiguration.java

```diff
@@ -66,6 +66,7 @@
   // Administration
   private static final String ADMIN_PREFIX = FLUO_PREFIX + ".admin";
   public static final String ADMIN_ACCUMULO_TABLE_PROP = ADMIN_PREFIX + ".accumulo.table";
+  public static final String ADMIN_HDFS_ROOT_PROP = ADMIN_PREFIX + ".hdfs.root";
```

Comment:
Talked about this offline.  Properties just used for interpolation in config file should be included in the class. I will remove.